src/lib/utils.js

```diff
@@ -181,6 +181,7 @@ function getRefinements(results, state) {
   forEach(state.numericRefinements, (operators, attributeName) => {
     forEach(operators, (values, operator) => {
       forEach(values, (name) => {
+        name = name + '';
```

Comment:
We chose `name` instead of `value`, I don't know why, but numeric refinements are supposed to be `Number`s, don't you think ?